src/main/java/Statique.java

```diff
@@ -7,7 +8,7 @@
 @Command(
         name = "statique",
         description = "A brand new static site generator.",
-        subcommands = {Init.class, Clean.class, Build.class, Serve.class})
+        subcommands = {Init.class, Clean.class, Build.class, Serve.class,Version.class})
```

Comment:
Désolée ça me trigger l'espace qui manque xD
```suggestion
        subcommands = {Init.class, Clean.class, Build.class, Serve.class, Version.class})
```

Reply:
fixé dans [bde337](https://github.com/dil-classroom/projet-egger_gehring_schaller_vogel/pull/41/commits/bde337c2ba1c4ddee29a397497811ff316b45e4e)

Reply:
fixé dans [bde337](https://github.com/dil-classroom/projet-egger_gehring_schaller_vogel/pull/41/commits/bde337c2ba1c4ddee29a397497811ff316b45e4e)